snapcraft/internal/project_loader/errors.py

```diff
@@ -84,6 +84,17 @@ def __init__(self, extension_name: str, base: str) -> None:
         super().__init__(extension_name=extension_name, base=base)
 
 
+class ExtensionUnsupportedClassicError(ProjectLoaderError):
+    fmt = (
+        "Failed to load extension {extension_name!r}: "
```

Comment:
As a suggestion, we have a rule in snapd where we use "cannot" strictly for every error message whenever that meaning is required. This makes the user experience just a tiny bit more consistent, instead of having "can't", "could not", "couldn't", "failed to", "unable to", etc. We also use lowercase on all error messages, so that they can be composed, as in "cannot do foo: the file doesn't exist", rather than Something: Like: This.

Reply:
This follows the pattern of all the other errors in the file. `Failed` is used several times in other messages, and all the other messages in the file are `Sentence case`.